src/web_app_skeleton/config/database.cr.ecr

```diff
@@ -7,7 +7,9 @@ Avram::Repo.configure do |settings|
     settings.url = ENV["DATABASE_URL"]? || Avram::PostgresURL.build(
       database: database_name,
       hostname: ENV["DB_HOST"]? || "localhost",
+      # Common default usernames may be "postgres", "root", or your system username
```

Comment:
I think if we say "common default usernames" we can remove "may be". Might be helpful to tell users who to get system username: `Common default usernames are "postgres", "root", or your system username (run 'whoami')`

Thoughts on that?